src/index.mjs

```diff
@@ -79,6 +82,11 @@ function rehypeAstroRelativeMarkdownLinks(options = {}) {
         webPathFinal += queryStringAndFragment;
       }
 
+      // add changes specific to our current environment
+      // webPathFinal =  webPathFinal.replace("/docs",  options.base || defaultBase)
+      // webPathFinal= webPathFinal.toLowerCase()
+      // webPathFinal=webPathFinal.replace(/ /g, "-");
```

Comment:
Is this default behaviour for astro? Will it automatically join links with spaces with "dashes"?

Reply:
Can confirm... 

```
arm64 [vernacchia:~/words-byvernacchia]$ ls -ahl src/content/blog/2024/02/
-rw-r--r--  1 vernacchia  staff   208B 29 Jan 14:54 test with SPACES.md
```

outputs like this

```
14:57:19   ├─ /blog/2024/02/test-with-spaces/index.html (+3ms)
```

Will put this back to how it was!